modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -12,26 +12,27 @@ The parameter defaults to [systemitem]``0``, which means unlimited.
 java.max_changelog_entries = 0
 ----
 
-If you set this parameter, it will come into effect only for new packages when they are synchronized.
-You might like to delete the cached data to remove older data.
+If you set this parameter (remember you need to restart the services with `spacewalk-service restart`), it will come into effect only for new packages when they are synchronized.
```

Comment:
```suggestion
If you set this parameter, it will come into effect only for new packages when they are synchronized.
After changing this parameter, restart services with ``spacewalk-service restart``.
```